android/app/build.gradle

```diff
@@ -94,23 +94,16 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3'
 
     // Lifecycle
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.7.0'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.8.3'
 
-    implementation "androidx.core:core-ktx:1.13.0"
-    implementation 'androidx.fragment:fragment-ktx:1.6.2'
+    implementation "androidx.core:core-ktx:1.13.1"
+    implementation 'androidx.fragment:fragment-ktx:1.8.1'
     implementation 'androidx.preference:preference-ktx:1.2.1'
 
-    implementation 'com.google.android.material:material:1.11.0'
+    implementation 'com.google.android.material:material:1.12.0'
 
     implementation 'com.github.tony19:logback-android:3.0.0'
 
-    implementation('commons-codec:commons-codec') {
-        version {
-            // use version 1.15 for compatibility reasons
-            strictly '1.15'
-        }
-    }
-
```

Comment:
This is not needed anymore as yubikit-android 2.6.0 uses own implementation of base32 and does not depend on commons-codec. 
We used commons-coded in yubikit-android only for b32 codec, however version 1.16 was not compatible with older Android APIs (and that is why we forced 1.15 in YA), so it was decided to implement our own b32 and not use the library at all. See https://github.com/Yubico/yubikit-android/pull/126